Packs/Jira/Integrations/JiraV3/JiraV3.yml

```diff
@@ -1460,6 +1460,56 @@ script:
       name: comment
     description: Use this command to create a link between two issues.
     name: jira-issue-to-issue-link
+  - arguments:
+    - description: The issue ID.
+      name: issue_id
+      required: true
+    description: Fetches questions and answers for forms tied to a Jira issue.
+    name: jira-issue-get-forms
+    outputs:
+    - contextPath: JiraForms.ID
+      description: The ID of the form.
+      type: Unknown
+    - contextPath: JiraForms.Issue
+      description: Issue the form belongs to.
+      type: Unknown
+    - contextPath: JiraForms.Questions
+      description: Questions for the form.
+      type: Unknown
+  - arguments:
+    - description: The user's key (also called ID).
+      name: key
+      required: false
+    - description: The user's username.
+      name: username
```

Comment:
same.

Reply:
Fixed, cloud clients will only use accountId and on prem will use the legacy params